src/components/AddBoxForm/AddBoxForm.jsx

```diff
@@ -59,21 +71,33 @@ const AddBoxForm = () => {
   });
 
   const [files, setFiles] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  const countryOptions = useMemo(() => countryList().getData(), []);
 
   const onSubmit = async data => {
     const formData = data;
     formData.date = formatDate(data.date);
     formData.launchedOrganically = formData.launchedOrganically === 'yes';
     formData.picture = files.length > 0 ? await uploadBoxPhoto(files[0]) : '';
+    formData.country = formData.country.value;
+    const [latitude, longitude] = await getLatLong(formData.zipcode, formData.country);
```

Comment:
There are times when this getLatLong just returns null values and I believe it is when the zipcode is not found in the country. If the lat/long is null, we should return an error here telling the user that the zipcode or country is invalid because if we cannot determine the lat/long coordinates, then the box will not show on the map.

Reply:
Yeah, so if you put 92777 for zipcode and US for the country, Nominatim API will return an empty list. But checking getLatLong isn't part of the yup validation so we wouldn't be able to check it recurringly nor would it be able to show an error message immediately. This is bc we are calling getLatLong only after all the yup validation has been checked and the user clicks submit. Is that okay? Lmk if this doesn't make sense. Also, if we do display an error message, would it just show up under the Zip Code field like any other error messages?